src/spring/azext_spring/custom.py

```diff
@@ -982,18 +984,71 @@ def config_repo_update(cmd, client, resource_group, name, repo_name,
 
     logger.warning("[2/2] Updating config server settings repo, (this operation can take a while to complete)")
     config_server_resource = models.ConfigServerResource(properties=config_server_properties)
-    return cached_put(cmd, client.begin_update_patch, config_server_resource, resource_group, name).result()
+    return cached_put(cmd, client.config_servers.begin_update_patch, config_server_resource, resource_group, name).result()
 
 
 def config_repo_list(cmd, client, resource_group, name):
-    config_server_resource = client.get(resource_group, name)
+    config_server_resource = client.config_servers.get(resource_group, name)
     config_server = config_server_resource.properties.config_server
 
     if not config_server or not config_server.git_property or not config_server.git_property.repositories:
         raise CLIError("Repos not found.")
 
     return config_server.git_property.repositories
 
+def config_create(cmd, client, service, resource_group):
+    try:
+        config_server_resource = client.config_servers.get(resource_group, service) 
+        if config_server_resource is not None:
+            raise CLIError("Config server '{}' already exists.".format(CONFIG_SERVER_DEFAULT_NAME))
+    except ResourceNotFoundException:
+        pass
+
+    properties = models.ConfigServerProperties()
+    config_server_resource = models.ConfigServerResource(properties=properties)
+    logger.warning("Creating config server")
+    return client.config_servers.begin_update_put(resource_group, service, config_server_resource)
+
+def config_delete(cmd, client, service, resource_group):
+    return client.config_servers.begin_delete(resource_group, service)
+
+def config_bind(cmd, client, service, resource_group, app):
```

Comment:
I didn't find the code for the `app` param in the `_param.py` file. Is it not needed or I missed it.